website/public/license/index.ejs.html

```diff
@@ -2181,162 +2181,165 @@ <h4>How to Apply These Terms to Your New Programs</h4>
+            <h4>Items That are Not Considered Part of a Modified Version</h4>
+            <p>
+              Works (including, but not limited to, modules and scripts) that
+              merely extend or make use of the Package, do not, by themselves,
+              cause the Package to be a Modified Version. In addition, such
+              works are not considered parts of the Package itself, and are not
+              subject to the terms of this license.
+            </p>
           </li>
-
           <li>
-            This license includes the non-exclusive, worldwide, free-of-charge
-            patent license to make, have made, use, offer to sell, sell, import
-            and otherwise transfer the Package with respect to any patent claims
-            licensable by the Copyright Holder that are necessarily infringed by
-            the Package. If you institute patent litigation (including a
-            cross-claim or counterclaim) against any party alleging that the
-            Package constitutes direct or contributory patent infringement, then
-            this Artistic License to you shall terminate on the date that such
-            litigation is filed.
+            <h4>General Provisions</h4>
+            <p>
+              Any use, modification, and distribution of the Standard or
+              Modified Versions is governed by this Artistic License. By using,
+              modifying or distributing the Package, you accept this license. Do
+              not use, modify, or distribute the Package, if you do not accept
+              this license.
+            </p>
+            <p>
+              If your Modified Version has been derived from a Modified Version
+              made by someone other than you, you are nevertheless required to
+              ensure that your Modified Version complies with the requirements
+              of this license.
+            </p>
+            <p>
+              This license does not grant you the right to use any trademark,
+              service mark, tradename, or logo of the Copyright Holder.
+            </p>
+            <p>
+              This license includes the non-exclusive, worldwide, free-of-charge
+              patent license to make, have made, use, offer to sell, sell,
+              import and otherwise transfer the Package with respect to any
+              patent claims licensable by the Copyright Holder that are
+              necessarily infringed by the Package. If you institute patent
+              litigation (including a cross-claim or counterclaim) against any
+              party alleging that the Package constitutes direct or contributory
+              patent infringement, then this Artistic License to you shall
+              terminate on the date that such litigation is filed.
+            </p>
           </li>
-
           <li>
-            <p>Disclaimer of Warranty:</p>
+            <h4>Disclaimer of Warranty:</h4>
             <p>
```

Comment:
I'm confused. Why is the first `<p>` disallowed but the second `<p>` allowed?

Reply:
For the elements I changed to `<h4>`. I assumed they were some level of subheading compared to the text that is was describing. A paragraph tag would work there if you would like me to change it to that. The main issue here was that the text I changed to `<h4>` tags were paragraph tags acting as headings outside of `<li>`, which is what was triggering the validation error.

Reply:
Here's what it should look like (approximately): https://github.com/quick-lint/quick-lint-js/blob/1a92c01a12d490427d297984ac64347af7cbe46f/website/public/npm.svg#L77-L102

The `<h4>`s should be *outside* the lists.

Reply:
Yeah I see what you are saying. The first `<p>` was not allowed because it was outside of a list item, and `<ol>` or `<ul>` elements can only have a `<li>` as a child. That`s why I had to wrap every section in a `<li>` tag. I did notice the numbering disappearing, so I figured I could add a nested list, the problem with that though was that the numbering wouldn't be continuous past other list items.

For example, this was giving validation error:
```html
<ol>
    <p>Permission for Use and Modification Without Distribution</p>
    <li>
         You are permitted to use the Standard Version and create and use
         Modified Versions for any purpose without restriction, provided that
         you do not Distribute the Modified Version.
    </li>
</ol>
```

This will not give validation error:
```html
<ol>
    <li>
        <h4>Permission for Use and Modification Without Distribution</h4>
        <p>
              You are permitted to use the Standard Version and create and use
              Modified Versions for any purpose without restriction, provided
              that you do not Distribute the Modified Version.
        </p>
    </li>
</ol>
```

Also, the reason the continuous numbering disappeared (photo below being the original) was because the `<li>` tags were being used anywhere it was needed along with other invalid children. I can probably just use css to target those items that need numbers with the ::before selector. I hope this makes sense.

![list_error](https://user-images.githubusercontent.com/59382692/121258405-78993480-c874-11eb-8409-6103784ddda0.png)

Reply:
> For example, this was giving validation error:

Ah, okay. Thanks for clarifying. I misunderstood the diff.

> I can probably just use css to target those items that need numbers with the ::before selector. I hope this makes sense.

Using CSS sounds awful! xD

> the problem with that though was that the numbering wouldn't be continuous past other list items.

Would something like this work:
```html
<h4>Permission for Use and Modification Without Distribution</h4>
<ol start="1">
<li>You are permitted to use ... the Modified Version.</li>
</ol>

<h4>Permissions for Redistribution of the Standard Version</h4>
<ol start="2">
<li>You may Distribute verbatim copies ... of the Package.</li>
<li>You may apply any bug fixes, ... subject to the Original License.</li>
</ol>

<h4>...</h4>
<ol start="4">
<li>...</li>
</ol>
```

Reply:
Yeah that works. Would you like me to remove the list default left padding? it makes it indent and not line up with its heading.

![list_new](https://user-images.githubusercontent.com/59382692/121262793-4e4a7580-c87a-11eb-97cf-36b850a8c36c.png)

Reply:
> Would you like me to remove the list default left padding? it makes it indent and not line up with its heading.

With CSS? Sounds unnecessary to me. I'm fine with how your screenshot looks.